.github/workflows/ci.yml

```diff
@@ -60,6 +61,28 @@ jobs:
         run: |
           echo "OPENMC_CROSS_SECTIONS=$HOME/endfb-vii.1-hdf5/cross_sections.xml" >> $GITHUB_ENV
 
+      - name: Install Additional Python Dependencies
+        shell: bash -l {0}
+        run: |
+          conda init bash
+          source ~/.bashrc
+          conda activate jupyter-actions
+          pip install vtk pyevtk xlwt swig
```

Comment:
Would it not work to put these in environment.yml?

Reply:
I'm not sure since it's running these through the `nbconvert` module in CI, but it's worth a go!

Reply:
Looks like that works just fine. Thanks for the suggestion!